frontend/src/admin/connections/components/Connections.js

```diff
@@ -156,7 +156,7 @@ const CustomRow = (props) => {
 			</TableCell>
 			{medium || small ? null :
 				<TableCell align={CONN_TABLE_COLUMNS[5].align} style={{padding: '6px'}}>
-					<Tooltip title={brokerConnection.status?.connected ? 'Disconnect' : 'Connect'}>
+					<Tooltip title={brokerConnection.status?.connected ? `Disconnect-${brokerConnection.id}` : `Connect-${brokerConnection.id}`}>
```

Comment:
Is this intentional?
The tooltip would look rather cryptic.

Reply:
Done.